lib/product.rb

```diff
@@ -0,0 +1,23 @@
+class Product
+  attr_reader :category, :name, :unit_price, :quantity, :is_hoarded
```

Comment:
you don't really ever call `.is_hoarded`, but you do call `.is_hoarded?` which is a method already. For that reason, I don't think you need to have `.is_hoarded` as an attr_reader -- not every attribute needs to have an attr_reader.